internal/test/wsu/wsu_test.go

```diff
@@ -85,9 +87,17 @@ func TestWSU(t *testing.T) {
 	require.NotEmptyf(t, playbookPath, "WSU_PATH environment variable not set")
 	require.NotEmptyf(t, clusterAddress, "CLUSTER_ADDR environment variable not set")
 
+	for _, vm := range framework.WinVMs {
+		runTestWSUAgainstVM(t, vm)
```

Comment:
nit: Do you want to run WSU tests on all the Windows VMs created?

Reply:
Same answer as before